### ISSUE TITLE
training terminates at 0%

### ISSUE THREAD

**Issue by pengcanon** [SJoJoK/3DGStream#30]
I'm training on my own multiview dataset and followed every step in the instruction. However, training simply just wouldn't proceed; it shows 0% progress and then terminates, with no error message given. Stepping through the code suggests that the termination occurs at running the first backpropagation; loss.backward() in train_frames.py. 

Any suggestions?

**Comment by pengcanon** [comment]
reducing image sizes and frames solved the problem. So it might be a memory issue
